sort.php

```diff
@@ -37,4 +37,8 @@ function bubbleSort($sort_array)
     return $sort_array;
 }
 
-echo implode("", bubbleSort(checkForNextLine(file('php://stdin'))));
\ No newline at end of file
+if(strpos($GLOBALS['argv'][1], '-') === false) {
```

Comment:
Please explain what exactly this condition does.

Reply:
It checks if the second command line argument value starts with a ``'-'`` character, not that it is e.g. ``-o`` or also ``--output``, in which case it is not a ``filename`` but an ``option``.